src/App.css

```diff
@@ -28,6 +28,34 @@
   color: #61dafb;
 }
 
+.title {
+  padding: 24px 24px 24px 24px;
+}
+
+.container {
+  display: flex;
+  height: 30px;
+  margin-bottom: 16px;
+  text-align: center;
+  padding: 0px 0px 0px 24px;
```

Comment:
so the `24px` padding is repetitive in both the `title` and `container`, what we can do is to have the padding in the outer container of both these element